spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +54,113 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before(:each) do
```

Comment:
Thanks @Haus - I'm going to take that as a maint suggestion for refactoring later